src/json.c

```diff
@@ -395,15 +529,26 @@ int json_nvmet_view_exports(struct NVMET_PROFILE *nvmet, struct NVMET_PORTS *por
 	json_array_append_new(array, ports_object);
 
 	root = json_pack("{s:o}", "targets", array);
-
-	printf("%s\n", json_dumps(root, 0));
-
-	json_decref(nvmet_array);
-	json_decref(ports_array);
-	json_decref(array);
+	char *jdumped = json_dumps(root, 0);
+//	if (jdumped != NULL) {
```

Comment:
Can we remove this commented code? It looks as if it was only used for troubleshooting.

Reply:
Yes, fixed!